deploy/crds/kustomize/policy.open-cluster-management.io_certificatepolicies.yaml

```
@@ -160,53 +177,77 @@ spec:
-                    to whether or not the policy is compliant
+                  description: CompliancyDetails reports the details related to whether
+                    or not the policy is compliant
                   properties:
                     message:
+                      description: Message is a human-readable summary of the compliance
+                        details.
                       type: string
                     nonCompliantCertificates:
+                      description: NonCompliantCertificates reports the total number
+                        of noncompliant certificates.
                       type: integer
                     nonCompliantCertificatesList:
                       additionalProperties:
-                        description: Cert contains its related secret and when it
-                          expires
+                        description: Cert reports a certificate's related secret and
+                          its parsed details.
                         properties:
                           ca:
+                            description: CA is a boolean reporting whether the certificate
+                              contains a CA.
                             type: boolean
                           duration:
                             description: |-
-                              A Duration represents the elapsed time between two instants
-                              as an int64 nanosecond count. The representation limits the
-                              largest representable duration to approximately 290 years.
+                              Duration is the total duration of the certificate by calculating the difference between its
+                              NotAfter and NotBefore values.
                             format: int64
                             type: integer
                           expiration:
+                            description: |-
+                              Expiration is the string representation of the certificate's expiration from the time of the
+                              report.
                             type: string
                           expiry:
                             description: |-
-                              A Duration represents the elapsed time between two instants
-                              as an int64 nanosecond count. The representation limits the
-                              largest representable duration to approximately 290 years.
+                              Expiry is the time.Duration representation of the certificate's expiration from the time of the
+                              report.
                             format: int64
                             type: integer
                           sans:
+                            description: Sans is a list of subject alternative names
+                              in the certificate.
```

Should "San" be displayed as an acronym? Or is "San" displayed like that from the policy?
```suggestion
                            description: SANS is a list of subject alternative names
                              in the certificate.
```

---

While you're right it is an acronym, unfortunately it was originally defined it in the code as `Sans`, so we should probably keep it like that to be consistent.